modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -220,6 +221,54 @@
             .nextUntil('.section-field').hide()
     }
 
+    /*
+     * Bind validation checks on available fields, triggered when
+     * the fields is blurred or changed
+     */
+    FormWidget.prototype.bindValidation = function () {
+        let fields = this.getFieldElements(),
```

Comment:
Current code base in October uses `var` not `let`, do you have an argument to switch?

Reply:
I prefer `let` as it restricts the scope of the variable to the specific block, and I believe IE11 supports it as well, but I've switched to `var` to keep it consistent.